core/arch/disassemble_shared.c

```diff
@@ -895,9 +887,13 @@ instr_disassemble_opnds_noimplicit(char *buf, size_t bufsz, size_t *sofar INOUT,
                          */
                         optype = 0;
                     });
+        bool is_mask = !dsts_first() && !instr_is_opmask(instr) && opnd_is_reg(opnd) &&
+            reg_is_opmask(opnd_get_reg(opnd));
+        print_to_buffer(buf, bufsz, sofar, is_mask ? "{" : "");
```

Comment:
(So this does work out nicely, right?)

Reply:
I actually thought it was more nicely encapsulated within internal_opnd_disassemble but yes, it prints the same way as before. At some point we may want to revisit it for Intel and ATT, i.e. print the mask after the destination like zmm {k}.